fees/liquidloans-io.ts

```diff
@@ -0,0 +1,10 @@
+import { CHAIN } from "../helpers/chains";
+import { liquityV1Exports } from "../helpers/liquity";
+
+export default liquityV1Exports({
+  [CHAIN.PULSECHAIN]: { 
+    troveManager: '0xD79bfb86fA06e8782b401bC0197d92563602D2Ab', 
+    redemptionEvent: 'event Redemption(uint256 _attemptedUSDLAmount, uint256 _actualUSDLAmount, uint256 _PLSSent, uint256 _PLSFee)',
```

Comment:
all the redemption events are the same? only the parameter name is different, so, we can use the default event?